build.sbt

```diff
@@ -253,6 +253,12 @@ def anyConds(conds: String*) = conds.mkString("(", " || ", ")")
 
 val faNpmAuthToken = "FONTAWESOME_NPM_AUTH_TOKEN" -> "${{ secrets.FONTAWESOME_NPM_AUTH_TOKEN }}"
 
+lazy val setupSbt = WorkflowStep.Use(
+  UseRef.Public("sbt", "setup-sbt", "v1"),
+  name = Some("Setup SBT"),
+  params = Map("sbt-runner-version" -> "1.10.2")
+)
```

Comment:
I think I prefer manually updating ci.yml and removing the workflow check. That way, when it is repaired upstream the workflows will regenerate and we don't have to undo anything.